src/libnnstreamer-edge/nnstreamer-edge-internal.c

```diff
@@ -1477,6 +1627,16 @@ nns_edge_release_handle (nns_edge_h edge_h)
         nns_edge_logw ("Failed to close AITT connection.");
       }
       break;
+    case NNS_EDGE_CONNECT_TYPE_CUSTOM:
+    {
+      NnsEdgeCustomDef *custom_h = (NnsEdgeCustomDef *) eh->custom_h;
+      int ret = custom_h->nns_edge_custom_close (eh->broker_h);
+      if (NNS_EDGE_ERROR_NONE != ret) {
+        nns_edge_loge ("Failed to stop MCF");
+      }
+      dlclose (eh->custom_h);
+      break;
+    }
```

Comment:
Good Code :100: 
Let's make it a function for the readability of the code. What do you think?

Reply:
👍 Thank you. 💯